lib/index.js

```diff
@@ -66,6 +67,11 @@ const Cache = (strapi) => {
         const pattern     = new RegExp(`^/${pluralize(model)}`);
         const keys        = await cache.keys();
 
+        if (!keys) {
+          warn('The request did not return any keys. Cache may be in an invalid state');
+          return next();
+        }
+
```

Comment:
Does ioredis sometime return `null` ? 

If so it might be worth moving this error management to the redis file, seems too specific to be in the main file.

Reply:
In this case ioredis does not return `null` for the keys, so I removed these. I just added a fallback value in case there is a timeout, so it doesn't add any noise here.